kratos/modeler/operation/voxel_mesher_operation.cpp

```diff
@@ -54,12 +54,26 @@ ModelPart& VoxelMesherOperation::CreateAndGetModelPart(std::string const& rFullN
     return mrModeler.CreateAndGetModelPart(rFullName);
 }
 
+std::size_t VoxelMesherOperation::GetNodeIndex(std::size_t I, std::size_t J, std::size_t K) const
+{
+    return mrModeler.mMeshingData.GetNodeIndex(I,J,K);
+}
+
+VoxelMeshGeneratorModeler::CartesianNodalData& VoxelMesherOperation::GetNodalData(std::size_t I, std::size_t J, std::size_t K) const
```

Comment:
```suggestion
VoxelMeshGeneratorModeler::CartesianNodalData& VoxelMesherOperation::GetNodalData(
    std::size_t I,
    std::size_t J,
    std::size_t K) const
```
Style minor comment.